.github/workflows/main.yml

```diff
@@ -0,0 +1,59 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Pytest with Flake8
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.8, 3.9]
```

Comment:
Should we just do the flake8 test on one python version?

Reply:
My main thought here is that it doesn't matter.  Flake8 is now decoupled from the tests.  It is currently set up so that if flake8 fails, the tests stop.  The upside to this is that you don't have to wait for an hour to see you have a style error.  

Long-term, I imagine adding a second workflow that only runs once for the documentation checks.  We could potentially move the flake8 check there to fully decouple from the multi-environment tests.